aiida_quantumespresso/cli/utils/options.py

```diff
@@ -4,15 +4,46 @@
 
 from aiida.cmdline.params import types
 from aiida.cmdline.params.options import OverridableOption
+from aiida.cmdline.utils import decorators
+from aiida.common import exceptions
 
 from . import validate
 
-STRUCTURE = OverridableOption(
-    '-s', '--structure', type=types.DataParamType(sub_classes=('aiida.data:structure',)), help='StructureData node.'
-)
+
+class PseudoFamilyType(types.GroupParamType):
+    """Subclass of `GroupParamType` in order to be able to print warning with instructions."""
+
+    @decorators.with_dbenv()
+    def convert(self, value, param, ctx):
+        """Convert the value to actual pseudo family instance."""
+        try:
+            group = super().convert(value, param, ctx)
+        except click.BadParameter:
+            try:
+                from aiida.orm import load_group
+                load_group(value)
+            except exceptions.NotExistent:  # pylint: disable=try-except-raise
+                raise
+            else:
+                raise click.BadParameter(  # pylint: disable=raise-missing-from
+                    f'`{value}` is not a supported pseudo potential family type.\nTo install a supported pseudo family,'
```

Comment:
```suggestion
                    f'`{value}` is not of a supported pseudopotential family type.\nTo install a supported pseudofamily,'
```

😄

Reply:
In my defense, you said there were some cases remaining in the README not anywhere else ;)